src/main/groovy/com/getkeepsafe/dexcount/System.groovy

```diff
@@ -0,0 +1,38 @@
+package com.getkeepsafe.dexcount
```

Comment:
pulled out some references to other classes only in SdkResolver.  E.g., the ability to pull down the Android SDK.  Totally out of scope for this plugin.
System.groovy is exactly the same.  I can also push the two files down a package too...that should avoid the System class conflict I think?